packages/frontend/src/components/Charts/CrabStrategyChart.tsx

```diff
@@ -128,9 +121,11 @@ export function CrabStrategyChart({
         // { data: convertPNLToPriceChart(shortSeries, startingETHPrice), legend: 'Short Squeeth (incl. funding)' },
       ]
     return [{ data: seriesRebalance, legend: 'PNL' }]
-  }, [vault, longEthPNL, shortEthPNL, seriesRebalance, getStableYieldPNL, longAmount, shortSeries])
+  }, [compoundSeries, getStableYieldPNL, longAmount, longEthPNL, seriesRebalance, shortEthPNL, shortSeries, vault])
```

Comment:
Do we use shortSeries?.length & compoundSeries?.length for dep array?

Reply:
Also other series in other dep arrays?

Reply:
In this case, all the series here have been memoized and are stable, so it's okay to put them in the deps array.